feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_ubus.c

```diff
@@ -10,6 +10,43 @@
 extern struct ev_loop *wifihal_evloop;
 static struct ubus_context *ubus;
 extern struct ev_loop *wifihal_evloop;
+extern void apc_state_set(struct blob_attr *msg);
+
+int ubus_set_required_signal(const char *if_name, int pval)
```

Comment:
Propagating existing bug:  rssi_ignore_probe_request and signal_connect should both be probe_resp_threshold from OVSDB, and signal_stay should be client_disconnect_threshold from OVSDB.

Therefore this function needs two value parameters and perhaps a clearer name.

Reply:
done